core/includes/classes/class-wp-rag-page-ai-configuration.php

```diff
@@ -18,16 +18,42 @@
 class Wp_Rag_Page_AiConfiguration {
 	const OPTION_NAME = 'wp_rag_ai_options';
 
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+	}
+
+	/**
+	 * Enqueue scripts for AI configuration page
+	 *
+	 * @param string $hook_suffix The current admin page
+	 */
+	public function enqueue_scripts( $hook_suffix ) {
+		// Only load on our specific admin page
+		if ( 'wp-rag_page_wp-rag-ai-configuration' !== $hook_suffix ) {
+			return;
+		}
```

Comment:
I'm not so sure, but calling `enqueue_scripts` from `class-wp-rag-run.php` like the following might be better:
https://github.com/mobalab/wp-rag/pull/43/files#diff-71f0fef9b9f2adf711e1e79dc766e21c5fc95fa87587d6c4427f481c57a7d277R275

If this approach is more popular and more desirable, let me know. In that case, I'll change the other PR to follow this style.